### ISSUE TITLE
The script only sync latest session

### ISSUE THREAD

**Issue by hucancode** [joshcai/leetcode-sync#48]
Hello, I have 3 sessions in my account, and after running the script. Only the latest session get synced.
By session, I mean Leetcode's session. On the right hand side of the [problems page](https://leetcode.com/problemset/all/), you can see it
![image](https://github.com/joshcai/leetcode-sync/assets/15852849/ab31daa8-036b-4d18-988e-915a05a8dba6)
Currently I have solved 90 Qs in this session. Previously I have solved 200+ and 400+ in older sessions
![image](https://github.com/joshcai/leetcode-sync/assets/15852849/74c79cd0-54bf-48c1-a491-e826a3cd88b4)
But only 90 Qs in the latest session get synced.
I have tried switching session, and then copy `LEETCODE_SESSION` and `csrftoken` then run the script. But seems like it doesn't work. 
```
[Fri, 22 Sep 2023 03:42:43 GMT] Getting submission from LeetCode, offset 0
[Fri, 22 Sep 2023 03:42:44 GMT] Default branch for hucancode/lc: master
[Fri, 22 Sep 2023 03:42:44 GMT] Syncing 0 submissions...
[Fri, 22 Sep 2023 03:42:44 GMT] Done syncing all submissions.
```

**Comment by hucancode** [comment]
I think I have found the cause. First time I ran the script, 90 newest problems got synced, the `lastTimestamp` point to the latest submission. After that all submission that come before that timestamp got ignored.

**Comment by Abhiram716** [comment]
> I think I have found the cause. First time I ran the script, 90 newest problems got synced, the `lastTimestamp` point to the latest submission. After that all submission that come before that timestamp got ignored. Work around for this is to sync submissions by chronological order if possible.
> 
> (Closed as a duplication of #17)

@hucancode Hey buddy, can allobrate the workaround please, I have 3 sessions in my leetcode too. I didn't understand sync submissions by chronological order if possible

**Comment by hucancode** [comment]
@Abhiram716 hi, let's say you have 2 sessions:
- switch leetcode session on the website to session 1
- make a new branch `git branch session-1`
- run the action for branch `session-1`
- switch leetcode session on the website to session 2
- make a new branch `git branch session-2`
- run the action for branch `session-2`
- now you have 2 new branches, each containing all submission for its session
- merge `session-1` and `session-2` to your main git branch
